ieeeconf/footer.tex

```diff
@@ -0,0 +1,31 @@
+% Bibliography
+
+\ifx\bibliocommand\undefined
+\else
+\printbibliography
+\fi
+
+\appendices
```

Comment:
I'm actually changing this in the next comment. the ending defined in the conference style guide is slightly different, and is being changed accordingly. It looks like I was a little preemptive with my PR.